core/stateful_precompile_test.go

```diff
@@ -543,8 +520,9 @@ func TestContractNativeMinterRun(t *testing.T) {
 	}
 
 	adminAddr := common.HexToAddress("0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC")
-	allowAddr := common.HexToAddress("0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B")
+	enabledAddr := common.HexToAddress("0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B")
 	noRoleAddr := common.HexToAddress("0xF60C45c607D0f41687c94C314d300f483661E13a")
+	testAddr := common.BigToAddress(common.Big2)
```

Comment:
Can we switch to using the same format of `common.HexToAddress(...)` to create each of the addresses?

Reply:
done